Sources/SwiftKuery/Index.swift

```diff
@@ -59,12 +59,8 @@ public struct Index {
             }
         }
         
-        var query = "CREATE INDEX "
-        
-        if isUnique {
-            query += " UNIQUE "
-        }
-        
+        var query = "CREATE \(isUnique ? "UNIQUE" : "") INDEX "
```

Comment:
MySQL fails on `CREATE INDEX UNIQUE`. It looks like `CREATE UNIQUE INDEX` is also the correct syntax for SQLite and PostGreSQL. So I made this change global. If there's some reason it is a problem for other databases, I can make it specific to MySQL.